jest-puppeteer.config.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   browserContext: 'incognito',
   browserPerWorker: true,
+  exitOnPageError: false,
```

Comment:
**suggestion:** worth adding a comment here that we handle this ourselves?

```suggestion
  // We handle errors ourselves in config/jest/environment/puppeteer.mjs
  exitOnPageError: false,
```

Reply:
Done